vector/src/main/res/values/strings.xml

```diff
@@ -2924,9 +2924,17 @@
     <!-- Location -->
     <string name="location_activity_title_static_sharing">Share location</string>
     <string name="location_activity_title_preview">Location</string>
+    <!-- TODO delete -->
```

Comment:
Thanks. Related lint issue about unused resourced could be ignore. Maybe add a lint ignore in the tag to please lint and not break develop once merged. I will update the recipe to delete strings if it works (or you can do it if you want to!).